src/screens/Vault/VaultMigrationController.tsx

```diff
@@ -248,7 +248,7 @@ function VaultMigrationController({
         }
 
         const allVaultIds = allVaults.map((vault) => vault.id);
-        const generatedVaultId = generateVaultId(signers, vaultScheme);
+        const generatedVaultId = generateVaultId([...signers, inheritanceSigner], vaultScheme);
```

Comment:
Maybe we can simply use vaultInfo.vaultSigners instead?
Given that the inheritance signer is an optional param, it won't be available for non MS-IK vaults.